thousandeyes/schemas.go

```diff
@@ -54,224 +54,6 @@ var schemas = map[string]*schema.Schema{
 				"agent_id": {
 					Type:        schema.TypeInt,
 					Description: "The unique ID for the ThousandEyes agent.",
-					Optional:    true,
```

Comment:
These fields are not needed, all of them, as they get removed here: https://github.com/thousandeyes/terraform-provider-thousandeyes/blob/main/thousandeyes/util.go#L164